doc/classes/Area3D.xml

```diff
@@ -116,7 +116,7 @@
 			The area's gravity vector (not normalized). If gravity is a point (see [member gravity_point]), this will be the point of attraction.
 		</member>
 		<member name="linear_damp" type="float" setter="set_linear_damp" getter="get_linear_damp" default="0.1">
-			The rate at which objects stop moving in this area. Represents the linear velocity lost per second. Values range from [code]0[/code] (no damping) to [code]1[/code] (full damping).
+			The rate at which objects stop moving in this area. Represents the linear velocity lost per second. Values range from [code]0[/code] (no damping) to [code]100[/code] (full damping).
```

Comment:
I don't think there's any upper limit. The 100 in the hint is just a reasonable max hint for the editor, you can still use more manually if you want.

Reply:
I just checked and there's no difference between 100 damping and 10000 damping. So 100 really seems to be max and anything more makes no sense. Also setting linear damping in RigidBody to 100 will make it stop completely.

Reply:
Damping calculations looks like:

```c++
real_t damp = 1.0 - step * linear_damp;

if (damp < 0) { // reached zero in the given time
	damp = 0;
}
```

With default physics fps (60) step should be `0.01666`, and dump values above `60` will have the same effect as `60`.

Reply:
```suggestion
			The rate at which objects stop moving in this area. Represents the linear velocity lost per second. Values range from [code]0[/code] (no damping) to [code]ProjectSettings.physics/common/physics_fps[/code] (full damping).
```

Reply:
And exactly the same code is used for `angular_damp`, but it's also documented as `0...1`.